declarations.d.ts

```diff
@@ -220,10 +220,18 @@ interface IAnalyticsService {
 	track(featureName: string, featureValue: string): IFuture<void>;
 }
 
+interface IAllowEmpty {
+	allowEmpty?: boolean
```

Comment:
![image axd](https://cloud.githubusercontent.com/assets/6683316/13455546/c8ef2454-e066-11e5-9edf-42b84d79f99a.jpg)

If we combine those two interfaces into one it would be misleading when using them inside methods which never reference one or more of these interface members